_posts/2021-01-19-bazel-4-0.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: posts
+title: "Bazel 4.0"
+authors:
+  - communikit
+---
+
+Bazel 4.0 is a major release and Bazel’s first LTS release. It contains new features and backwards incompatible changes.
+
+## New features
+
+* Starlark-defined flags can now be shorthanded using `--flag_alias`.
+* Support has been added for [building with a Java 15 toolchain](https://github.com/bazelbuild/bazel/pull/12168).
+* Targets incompatible with the platform [can now be skipped](https://github.com/bazelbuild/bazel/issues/3780). See the [Platforms](https://docs.bazel.build/versions/master/platforms.html) documentation for more details.
```

Comment:
In general, it's good to end with the link/ call to action. Flip this sentence around to be:
For more details, see the [Platforms documentation](https://docs.bazel.build/versions/master/platforms.html).

Reply:
Done.

Reply:
Done.